From d224e0a2b4e713808f6bc307723e71a2d13869a9 Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Sun, 2 May 2021 17:12:58 -0700
Subject: [PATCH] Update Version to 1.1.1

This might already be out of date. but I figured this was better than nothing.
---
 src/version.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/version.h b/src/version.h
index 69ebd50..474585c 100644
--- a/src/version.h
+++ b/src/version.h
@@ -5,9 +5,9 @@
 #ifndef SRC_VERSION_H_
 #define SRC_VERSION_H_
 
-#define RCEDIT_MAJOR_VERSION 0
-#define RCEDIT_MINOR_VERSION 2
-#define RCEDIT_PATCH_VERSION 0
+#define RCEDIT_MAJOR_VERSION 1
+#define RCEDIT_MINOR_VERSION 1
+#define RCEDIT_PATCH_VERSION 1

 #ifndef RCEDIT_TAG
 # define RCEDIT_TAG ""
